### DISCUSSION TITLE
How to track papers labeled as irrelevant in ASReview: SAFE produre

### DISCUSSION THREAD

**Discussion by ShuiqingHuang** [asreview/asreview#1673] [Q&A]
We are consider doing the review using SAFE procedure, the Phase 4 requires us to use the previous data labeled as irrelevant to screen again for quality check. However, when I exported the dataset in excel format, I could only see the study included and view_ranking. I would like to know if I want to specify the records as labeled as irrelevant but not the unseen data, how can I get this part of data?

<img width="381" alt="20240205121724" src="https://github.com/asreview/asreview/assets/158477037/2decff2b-c231-4026-b11a-87f07da95936">

**Comment by jteijema** [comment]
Hi @ShuiqingHuang 

Did you check the bottom of the file? The ASReview ranking for irrelevant files is naturally very low, and it might be that those files you labeled as irrelevant are at the bottom.

^JT

**Reply by Rensvandeschoot** [reply]
Indeed! The file is ordered as follows:

1. First, you will see all relevant records you have seen in the order they were shown during the screening process.
2. Then, all records not seen during the screening are ordered from most to least relevant according to the last iteration of the model.
3. At the bottom of the file you will find all non-relevant records presented in the order these are shown during the screening process.

see, https://asreview.readthedocs.io/en/latest/progress.html#export-results
